temp.py

```diff
@@ -0,0 +1,8 @@
+def reset_files(filepath, file_type="txt"):
+    with open(filepath, "w") as output_file:
```

Comment:
The function could be made more robust by checking if the file exists before trying to open it. This could prevent runtime errors. [medium]